### PR TITLE
Bug Fix: created new div and p tag styles and wrap it on the workspace member as container 

### LINKED ISSUE #7574: Crop object name in New relation field Settings
# Current behavior

CleanShot 2024-10-10 at 17 38 15

# Desired Behavior

Workspace member should be cropped and a tooltip should display the full text:

`Workspace Memb....`

### PR DESCRIPTION
**Pull request by HamzaVillage**
Hello,
Hope you are doing well.I created a special style for the text to make sure it stays in one line and wont exceed the width if the text width will be more then 80px it will ecplise and set ... at the end of the text.

I created these 2 styles variables and wrap my text in these styles 
StyledObjectSummary 
StyledEllipsisParagraph 

Fixes #7574 

#Screens Shots
Screenshot 2024-10-10 at 10 58 04 PM

Screenshot 2024-10-10 at 10 58 20 PM

### CODE REVIEW

File: packages/twenty-front/.env.example
Comment (ehconitin) [inline packages/twenty-front/.env.example:1]: Could you restore this .env.example  :)

File: packages/twenty-front/src/modules/settings/data-model/fields/forms/components/SettingsDataModelFieldSettingsFormCard.tsx [hunk 1]
```diff
@@ -129,8 +129,8 @@ export const SettingsDataModelFieldSettingsFormCard = ({
   fieldMetadataItem,
   objectMetadataItem,
 }: SettingsDataModelFieldSettingsFormCardProps) => {
+  console.log("fieldMetadataItem",fieldMetadataItem.type)
```
Comment (ehconitin) [inline packages/twenty-front/src/modules/settings/data-model/fields/forms/components/SettingsDataModelFieldSettingsFormCard.tsx:132]: remove all logs please

File: packages/twenty-front/src/modules/settings/data-model/fields/forms/relation/components/SettingsDataModelFieldRelationSettingsFormCard.tsx [hunk 2]
```diff
@@ -75,6 +75,7 @@ export const SettingsDataModelFieldRelationSettingsFormCard = ({
   const relationType = watchFormValue('relation.type', initialRelationType);
   const relationTypeConfig = RELATION_TYPES[relationType];
 
+  console.log("relationTypeConfig.imageSrc",initialRelationFieldMetadataItem)
```
Comment (ehconitin) [inline packages/twenty-front/src/modules/settings/data-model/fields/forms/relation/components/SettingsDataModelFieldRelationSettingsFormCard.tsx:78]: same

File: packages/twenty-server/.env.example
Comment (ehconitin) [inline packages/twenty-server/.env.example:1]: restore please :)!

**Review by ehconitin — COMMENTED** [review]
Left some comments

File: packages/twenty-server/.env.example
Comment (SyedHamzaHussain000) [reply packages/twenty-server/.env.example:1]: sure sure

**Comment by charlesBochet** [comment]
@SyedHamzaHussain000 still missing the twenty-server .env.example :)

**Comment by SyedHamzaHussain000** [comment]
@charlesBochet Sorry about that. I fixed that and push the new commit please check Thankyou

**Comment by charlesBochet** [comment]
@SyedHamzaHussain000 why are we getting rid of the icon?

File: packages/twenty-front/src/modules/settings/data-model/objects/SettingsDataModelObjectSummary.tsx [hunk 3]
```diff
@@ -18,11 +18,12 @@ const StyledObjectSummary = styled.div`
   justify-content: space-between;
 `;
 
-const StyledObjectName = styled.div`
-  align-items: center;
-  display: flex;
-  font-weight: ${({ theme }) => theme.font.weight.medium};
-  gap: ${({ theme }) => theme.spacing(1)};
+const StyledEllipsisParagraph = styled.p`
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/settings/data-model/objects/SettingsDataModelObjectSummary.tsx:21]: name is not clear:  I prefer the existing StyledObjectName :)

File: packages/twenty-front/src/modules/settings/data-model/objects/SettingsDataModelObjectSummary.tsx [hunk 4]
```diff
@@ -18,11 +18,12 @@ const StyledObjectSummary = styled.div`
...
+  max-width: 80px;      
+  white-space: nowrap;  
+  overflow: hidden;     
+  text-overflow: ellipsis; 
+  margin: 0;
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/settings/data-model/objects/SettingsDataModelObjectSummary.tsx:26]: let's use a div and not a p, so we don't have to add margin: 0

File: packages/twenty-front/src/modules/settings/data-model/objects/SettingsDataModelObjectSummary.tsx [hunk 5]
```diff
@@ -37,10 +38,9 @@ export const SettingsDataModelObjectSummary = ({
 
   return (
     <StyledObjectSummary className={className}>
-      <StyledObjectName>
-        <ObjectIcon size={theme.icon.size.sm} stroke={theme.icon.stroke.md} />
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/settings/data-model/objects/SettingsDataModelObjectSummary.tsx:41]: why are we removing the icon?

**Comment by charlesBochet** [comment]
Also the CIs are not passing, a priori because of lint issues

**Comment by charlesBochet** [comment]
Taking over this one

**Review by charlesBochet — APPROVED** [review]

**Pull request merged**